include/z64save.h

```diff
@@ -212,7 +214,10 @@ typedef struct {
     /* 0x1420 */ s16 worldMapArea;
     /* 0x1422 */ s16 sunsSongState; // controls the effects of suns song
     /* 0x1424 */ s16 healthAccumulator;
-} SaveContext; // size = 0x1428
+#ifdef FW_SPLIT_AGE
+    /* 0x1428 */ FaroresWindData fw_secondary;
```

Comment:
I'd rather this be next to the original variable. I know this will change the structure of SaveContext but that is going to have to happen regardless, so if that breaks any speedrunning glitches or something it is a necessary evil for this project, imo.

Reply:
tbh I was just worried about shifting savecontext raising issues, I believe it's still a bit weird how it's handled in decomp, but yea it would be better next to the original variable

Reply:
If there is an alternate struct in z_sram.c for this still, then just update that one too for now. I'm going to have to go through and fix the save context issues soon.

Reply:
(on a side note: I don't think there's any glitch that requires the savecontext, except SRM but I expect this to be patched on this repo as it can be problematic in mods)